### DISCUSSION TITLE
Creating a dosing compartment in Mobi (for subcutaneous dosing)

### DISCUSSION THREAD

**Discussion by jgg1822** [Open-Systems-Pharmacology/Forum#892] [Questions & Problems] [answered]
Hi all,

I've followed the instructions in the link below to create a new compartment within PK-Sim to replicate subcutaneous dosing by copying the fat compartment and making it 1mL in volume.

https://ascpt.onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1002%2Fpsp4.12300&file=psp412300-sup-0001-suppinfo01.pdf

Now I am trying to change the dosing compartment from venous blood to this new compartment (SQcmpt), as I've copied over IV bolus dosing in the simulation. I think that I basically need to implement #3 and #5 in the comment linked below, but I cannot see where to change the path for the 'changed molecule', referenced in #3?

https://github.com/Open-Systems-Pharmacology/MoBi/issues/246#issuecomment-370696224

Thanks in advance for any help!

**Comment by PavelBal** [comment]
Hi,

you must change the "Changed entity path":
![image](https://user-images.githubusercontent.com/25903040/116983654-a7971600-acca-11eb-8129-aae7bba469f8.png)

Click on the three dots, navigate to the SQcmpt, and select the molecule there.

Hint: You cannot directly edit the path, so you cannot use the generic "MOLECULE", which means that the changed molecule depends on the molecule you are administering. You have either to select a specific molecule or, in case you want your Administartion protocol to work for different molecules, export the Application to pkml, open the pkml with a text editor, and replace the name of the molecule in the path with "MOLECULE". After that, you can load the pkml as an application in MoBi.

**Comment by jgg1822** [comment]
Thank you for the quick reply! I tried implementing the steps you described, but I'm getting an error that the changed entity path is not recognized. Am I incorporating your suggestions correctly, or maybe something else is going on? My MoBi file is attached.

[SQ test.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/6423226/SQ.test.zip)

**Comment by PavelBal** [comment, marked as answer]
If you open your simulation and navigate to the SC-organ, you will see that no molecules are present in the subcompartments.

1) Open Molecules Start Values Building Block (BB)
2) Click on "extend". You will get new entries for all molecules of the project in the new organ. By default, all molecules are present there. I would only set the molecule you want to dose as "Is Present" in this organ.

![image](https://user-images.githubusercontent.com/25903040/117053585-6f6af400-ad19-11eb-8ee7-48d777e46f91.png)


At this stage, you will get an error that a paramter "Partition coefficient (interstitial/water) for the molecule is not found. Please refer to the manual you cite in the previous post how to add this parameter to the model.
